src/components/wallet/card/wallet-card.tsx

```diff
@@ -7,6 +7,7 @@ import CheckIcon from '@assets/svg/card/ic-check.svg';
 import { useEffect, useState, useRef } from 'react';
 import Text from '@core/text/text';
 import SVG from 'react-inlinesvg';
```

Comment:
@maryia-deriv Something out of the scope of this PR. Why do we use `react-inlinesvg`?

Reply:
@balakrishna-binary it has a MIT license and suits my purpose to create an inline svg where I can change it's children's exact attributes (fill/stroke) in order to update WalletCard single svg background colors on the fly instead of using several svg files that differ in colors only. 🙂

Reply:
@maryia-deriv we can actually use CSS variables in SVG too to change color dynamically. We do that to SVGs in `deriv-app`.  Is there any other reason for using `inlinesvg`?
I'm actually moving this component to `deriv-app`. Including `react-inlinesvg` package increases the bundle size 😅 
If there is no other reason, I think I'll move this component and update the SVGs to use CSS variables.

Reply:
@balakrishna-binary My intention was to avoid changing the svg file manually (in order to set fill="var(--fill-color)" everywhere we want to apply -fill-color on the children we need) with the aim of simplifying the replacement the svg file with a new one in future. Besides that, no other reason. As agreed: Thank you so much for taking care of updating the SVGs to use CSS variables 😊